src/index.js

```diff
@@ -1,130 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import ReactDOM from 'react-dom/client';
+import Board from './Board.js';
+import { Status } from './Info.js';
+import { Moves } from './Info.js';
+import { calculateWinner } from './Utils.js';
 import './index.css';
 
-function Square(props) {
-  return (
-    <button className='square' onClick={props.onClick}>
-      {props.value}
-    </button>
-  );
-}
-
-function Board(props) {
-  function renderSquare(i) {
-    return <Square value={props.squares[i]} onClick={() => props.onClick(i)} />;
-  }
-
-  return (
-    <div>
-      <div className='board-row'>
-        {renderSquare(0)}
-        {renderSquare(1)}
-        {renderSquare(2)}
-      </div>
-      <div className='board-row'>
-        {renderSquare(3)}
-        {renderSquare(4)}
-        {renderSquare(5)}
-      </div>
-      <div className='board-row'>
-        {renderSquare(6)}
-        {renderSquare(7)}
-        {renderSquare(8)}
-      </div>
-    </div>
-  );
-}
-
-class Game extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      history: [{ squares: Array(9).fill(null) }],
-      stepNumber: 0,
-      xIsNext: true,
-    };
-  }
+function Game() {
+  const [history, setHistory] = useState([{ squares: Array(9).fill(null) }]);
+  const [stepNumber, setStepNumber] = useState(0);
+  const [xIsNext, setXIsNext] = useState(true);
```

Comment:
there is no need for this state as you can do `const xIsNext = index % 2 === 0`

Reply:
That's a good call. Thank you @meesvandongen

Reply:
@cabrera-carlos I will recommend you to use the `index.js` as the entry point of the application (usually, to wrap your entire application (App.js) with different providers like Apolo, BrowserRouter, etc) and use the `App.js` component (not being used at all) to render the `Game` component.

What I will do is:
1. Move the `Game` component to its on file
2. import `Game` component into `App`
3. Render `App` in the `createRoot` function like:
`const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(<App/>);`